POS/Integration/05_ConfigureGlobalSettings.md

```diff
@@ -1,25 +1,22 @@
 [!!Create a store in POS](./06_CreateStore.md)
 [!!User interface Global settings](../UserInterface/02a_GlobalSettings.md)
-[!!User interface DataHub ETL](../../DataHub/UserInterface/02d_ETL.md)
+[!!User interface DataHub MAPPINGS](../../DataHub/UserInterface/02a_Mappings.md)
 
-[comment]: <> (add link to Order management module if available)
 
 # Configure the global settings for POS
 
-Venduo POS provides many settings to configure the POS system individually.
-
-For detailed information about the specific settings, see [Global Settings](../UserInterface/02a_GlobalSettings.md).
+POS offers many settings to customize the POS system to your specific needs. For detailed information about the specific settings, see [Global Settings](../UserInterface/02a_GlobalSettings.md).
 
 Before creating a new store in the POS system, you should configure the following settings:
 
-- [Configure a default customer](#configure-a-default-customer) whose billing address will be added to all future POS bills to establish a connection to the *Order management* module.
-- [Configure the VAT-ID in POS](#configure-the-VAT-ID-in-POS) for the printing on receipts, which is mandatory in most countries.
+- Configure a default customer whose billing address will be added to all future POS bills to establish a connection to the *Order Management* module. For detailed information, see [Configure a default customer](#configure-a-default-customer).
+- Configure the VAT ID for POS for the printing on receipts, which is mandatory in most countries. For detailed information, see [Configure the VAT ID in POS](#configure-the-vat-id-in-pos).
```

Comment:
Woanders VAT-ID. Bitte vereinheitlichen (ohne Bindestrich).
In der Doku und im Internet üblicherweise ohne Bindestrich.